rpc.go

```diff
@@ -113,6 +114,28 @@ type libvirtError struct {
 	Level    uint32
 }
 
+func (e libvirtError) Error() string {
+	return fmt.Sprintf("%s, code: %d, domain id: %d", e.Message, e.Code, e.DomainID)
```

Comment:
I don't think that displaying the error numbers here is helpful to the end user. This should just `return e.Message.`